test/integration/ferc1_eia_train_test.py

```diff
@@ -70,7 +70,7 @@ def ferc1_eia_training_data():
         ),
         pytest.param(
             ["True"],
-            [2019],
+            [2021],
             ["2_2020_plant_owned_195"],
             ["f1_steam_2020_12_2_0_1"],
             [18],
```

Comment:
is it a problem that the year here is different than the year in the two test ids?

Reply:
No, that's the point of the test - to fail when there is a discrepancy in years. I just wanted to update the year.